web-app/CKS.Web.Test/IntegrationTests/SearchTests.cs

```diff
@@ -11,19 +18,67 @@ public class SearchTests : TestBase
 		public SearchTests(ITestOutputHelper output) : base(output)
 		{ }
 
-		[Obsolete("Please replace this test with a real test when the search endpoint is setup")]
+
+		private HttpClient SetupClientWithFakeSearchProvider(string fakeSearchResultsFilePath = null)
+		{
+			var searchResults = new SearchResults();
```

Comment:
Maybe this stuff sits better in the WebApplication Factory? Ie as a WithFakeSearchProvider method or something, like we do here: https://github.com/nice-digital/MAS/blob/master/lambda/MAS.Tests/Infrastructure/MASWebApplicationFactory.cs#L86. As in it's infrastructure-type setup which is pretty generic and could be re-used across tests.

If we do keep it here though might make more sense to set it up in the constructor rather than having a private method:

> xUnit.net creates a new instance of the test class for every test that is run, so any code which is placed into the constructor of the test class will be run for every single test.